VK/VK/Presentation/Profile/ViewControllers/ProfileViewController.swift

```diff
@@ -35,6 +35,7 @@ final class ProfileViewController: UIViewController {
     // MARK: - Private Properties
 
     private var index = 0
+    private var session = NetworkService()
```

Comment:
```suggestion
    private var networkService = NetworkService()
```
у тебя где-то название service а тут session
лучше сделать везде одинаково